compiler/rustc_typeck/src/collect.rs

```diff
@@ -1,4 +1,3 @@
-// ignore-tidy-filelength
 //! "Collection" is the process of determining the type and other external
 //! details of each item in Rust. Collection is specifically concerned
 //! with *inter-procedural* things -- for example, for a function
```

Comment:
this causes CI to fail
https://github.com/rust-lang/rust/pull/94161#issuecomment-1046053573

Reply:
Something confusing happened https://github.com/rust-lang/rust/commit/7ca1c48bbb6d300bda665a59b7ad226bc71e7be9#diff-ac1e6e554bac799e1e2d3886bea8fdc0d0bf04b20d33df59f5a8e92c41987ac6

I've just rebased this so it should pass CI now - though the version number still needs updating if this isn't going into 1.60 beta as Amanieu said.